src/app/app-common/include/OpenKneeboard/DoodleSettings.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
+
+namespace OpenKneeboard {
+struct DoodleSettings final {
+  uint32_t DrawRadius = 50;
+  uint32_t EraseRadius = 150;
+  bool FixedDraw = false;
+  bool FixedErase = true;
+  // TODO: stroke colour?
```

Comment:
Doesn't matter for a comment though FYI: despite growing up in England, I'm generally using US English here. DirectX and the windows API consistently use 'color', so using 'color' means we need to mix different spellings in the same code